taxonomy-editor-frontend/src/pages/allentries/EntriesList.jsx

```diff
@@ -0,0 +1,84 @@
+import { createTheme, ThemeProvider, Typography } from "@mui/material";
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import EditIcon from '@mui/icons-material/Edit';
+import IconButton from '@mui/material/IconButton';
+import { useNavigate } from "react-router-dom";
+
+const EntriesList = ({ nodes, title }) => {
+    const navigate = useNavigate();
+    const handleClick = (event, id) => {
+        event.preventDefault();
+        navigate('/entry/'+id);
+    }
+    const theme = createTheme({
+        typography: {
+            fontFamily : 'Plus Jakarta Sans',
+        },
+    });
+    return ( 
+        <ThemeProvider theme={theme}>
+            <div className="entry">
+                <Typography sx={{mb: 1, mt:2, ml: 2}} variant="h4">
+                    List of nodes in {title} Taxonomy:
+                </Typography>
+                <div className="entry-preview">
+                    <TableContainer sx={{ml: 2}} component={Paper}>
+                        <Table sx={{ width: 500 }}>
+                            <TableHead>
+                            <TableRow>
+                                <TableCell align="center">
+                                <Typography variant="h6">
+                                    #
+                                </Typography>
+                                </TableCell>
+                                <TableCell align="center">
+                                <Typography variant="h6">
+                                    Nodes
+                                </Typography>
+                                </TableCell>
+                                <TableCell align="center">
+                                <Typography variant="h6">
+                                    Action
+                                </Typography>
+                                </TableCell>
+                            </TableRow>
+                            </TableHead>
+                            <TableBody>
+                                {nodes.map((node, index) => (
+                                    <TableRow
+                                    key={node[0].id}
+                                    sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
+                                    >
+                                        <TableCell align="center" component="th" scope="row">
+                                            <Typography variant="subtitle1">
+                                                {index+1}
```

Comment:
is this needed?

Reply:
@nobeeakon It was just a way to keep track of the number of entries in the table. It isn't necessary, but the table seemed to be clear with it. Do you think it needs to be removed?

Reply:
yeah @aadarsh-ram, not sure on that... I didn't run the code so I'm not completly sure how this looks like. On the other side, I think this is more of a UX decision, if this info is providing the user some additional/useful information it is good to keep. But, if the index is not providing any info the user actually needs or uses I'd remove it as doing so would save some space. If what you want is to have just a counter what about adding a small legend at the bottom that says how many rows the table has?

Reply:
Yep @nobeeakon, I think that would work. Didn't think about it that way. I'll make the necessary changes.